### PR TITLE
Slack message chunking

### PR DESCRIPTION
**Pull request by AmeyMore98**
Add support to batch slack updates

### CODE REVIEW

File: cowin_notifier/config.py [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+from pydantic.main import ModelMetaclass
 
 
-class Config(BaseSettings):
+class EnvironmentVariable(BaseSettings):
     SLACK_WEBHOOK: str
     MIN_AGE_LIMIT: int = 45
 
 
+class Config(EnvironmentVariable):
```
Comment (saintlyzero) [inline cowin_notifier/config.py]: Why do we need this `__init__()`?
Our env variables are pretty straightforward without any complex datatypes
I think `environs` is an overkill, `Pydantic` `BaseSettings` suffice our needs

File: cowin_notifier/main.py [hunk 2]
```diff
@@ -42,16 +47,24 @@ async def init_db() -> None:
...
+    await CowinNotifier().watch_and_notify()
+
+
+@app.on_event("startup")
+@repeat_every(
+    seconds=Constants.DYNO_INACTIVITY_DELAY,
+)
+def keep_dyno_alive() -> None:
+    """Run every 25 minutes to keep Heroku dyno alive"""
+    logger.info("We're stayin' alive, stayin' alive Ah, ha, ha, ha, stayin' alive")
```
Comment (saintlyzero) [inline cowin_notifier/main.py:62]: Seriously this made me chuckle :joy:

File: cowin_notifier/utils.py [hunk 3]
```diff
@@ -112,3 +112,18 @@ def get_checkbox(self, options: List[dict], action_id: str) -> dict:
...
+
+    Args:
+        _list (list): List of items
+        chunk_size (int): Max chunk size
+
+    Yields:
+        Iterator[list]: One chunk
+    """
+    for i in range(0, len(_list), chunk_size):
+        yield _list[i:i + chunk_size]
```
Comment (saintlyzero) [inline cowin_notifier/utils.py:129]: Ahh, generators :100:

File: cowin_notifier/config.py (hunk 1 above)
Comment (AmeyMore98) [reply cowin_notifier/config.py]: My bad, didn't know `BaseSettings` could be used like that 🙈. Pushed a fix.

File: cowin_notifier/main.py (hunk 2 above)
Comment (AmeyMore98) [reply cowin_notifier/main.py:62]: 😂

**Pull request merged**